src/_main_/utils/common.py

```diff
@@ -106,11 +106,14 @@ def rename_fields(args, pairs):
   return args
 
 
-def serialize_all(data, full=False):
+def serialize_all(data, full=False, **kwargs):
```

Comment:
I'd much prefer the following.
Define an enum like this:

```
from enum import Enum

class SerializationType(Enum):
    SIMPLE = "s"
    FULL = "f"
    MEDIUM = "m"
    # ... we can add as many as we want here
```

Then over here we can simplify this as 
```suggestion
def serialize_all(data, serialization_type: SerializationType= SerializationType.FULL):
```

Then you can go on and do any gymnastics in the models when we call json

Reply:
can make this change later.  For now feature flags returned in simple_json for communities